src/components/RulesSelector.jsx

```diff
@@ -27,19 +29,14 @@ const makeListItems = (filters, onRuleSelected) => (
 const RulesSelector = ({ filters, onRuleSelected }) => {
   const listItems = makeListItems(filters, onRuleSelected);
   return (
-    <div>
+    <div className="rule-selector-container">
```

Comment:
We're using both of those methods in the project, this isn't new. Also, it's more conducive for development since you don't have to reload the app to see your changes.